roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -139,6 +139,10 @@ networkConfig:
 {% endif %}
 # serviceNetworkCIDR must match kubernetesMasterConfig.servicesSubnet
   serviceNetworkCIDR: {{ openshift.common.portal_net }}
+  externalIPNetworkCIDRs:
+{%- for openshift_master_external_ip_network_cidr in openshift.master.external_ip_network_cidrs %}
+      - "{{ openshift_master_external_ip_network_cidr }}"
+{%- endfor %}
```

Comment:
Rather than taking this approach, it may be better to test if external_ip_network_cidrs is a list and if it's length is greater than 0, and then use the to_padded_yaml filter:
```
{%- if external_ip_network_cidrs in openshift.master and isinstance(openshift.master.external_ip_network_cidrs, list) and openshift.master.external_ip_network_cidrs | count > 0 %}
  externalIPNetworkCIDRs: "{{ openshift_master_external_ip_network_cidr | to_padded_yaml(1,2) }}"
{%- endif %}
```